presto-main/src/test/java/com/facebook/presto/operator/aggregation/AggregationTestUtils.java

```diff
@@ -382,4 +402,16 @@ public static Object getGroupValue(GroupedAccumulator groupedAggregation, int gr
     {
         return IntStream.range(start, start + length).asDoubleStream().toArray();
     }
+
+    private static void assertWithErrorMessage(BiConsumer<Object, Object> equalAssertion, Object left, Object right, String errorMessage)
```

Comment:
it might be cleaner to change `equalAssertion` to a function that takes 3 arguments; or, perhaps, change it to a function that return a boolean indicating whether the two objects are equal and use something like `assertTrue(.., String.format("Expected %s, but got %s", expected, actual)`